avalon/tools/cbsceneinventory/app.py

```diff
@@ -527,15 +643,19 @@ def refresh(self):
         with preserve_expanded_rows(tree_view=self.view,
                                     role=self.model.UniqueRole):
             with preserve_selection(tree_view=self.view,
-                                    role=self.model.UniqueRole):
-                self.model.refresh()
+                                    role=self.model.UniqueRole,
+                                    current_index=False):
```

Comment:
I changed the `current_index` to `False`, this is for preserving the selection of multiple items.